src/blocks/TitleParagraphBlockBrandPivot.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {
+  ContentWrapper,
+  MOBILE_BP_DOWN,
+  SectionWrapper,
+} from '../components/blockHelpers'
+import {
+  BrandPivotBaseBlockProps,
+  MarkdownHtmlComponent,
+} from './BaseBlockProps'
+
+const Headline = styled('h4')({
+  textTransform: 'uppercase',
+  fontSize: '1rem',
+  margin: 0,
+  [MOBILE_BP_DOWN]: {
+    textAlign: 'center',
+  },
+})
+
+const Paragraph = styled('div')({
+  display: 'block',
+  maxWidth: '100%',
+  width: '100%',
+  fontSize: '2rem',
+  lineHeight: '125%',
+
+  [MOBILE_BP_DOWN]: {
+    maxWidth: '100%',
+    display: 'block',
+    fontSize: '1.25rem',
+    textAlign: 'center',
+  },
+
+  '> :first-child': {
+    marginTop: '1rem',
+  },
+
+  '> p:last-child': {
+    marginBottom: 0,
+  },
+})
+
+interface TitleParagraphBlockInterface extends BrandPivotBaseBlockProps {
```

Comment:
A minot, but let's stick to props for consistency with the other components 
```suggestion
interface TitleParagraphBlockProps extends BrandPivotBaseBlockProps {
```